.travis.yml

```diff
@@ -1,24 +1,42 @@
-language: python
-python:
-  - "2.7"
+language: generic
+sudo: false
 env:
-  - REQUIREMENTS=dev_requirements.txt
-  - REQUIREMENTS=release_requirements.txt
-  - REQUIREMENTS=enaml_0_8_9_requirements.txt
-virtualenv:
-  system_site_packages: true
+  global:
+    - INSTALL_EDM_VERSION=1.9.2
+      PYTHONUNBUFFERED="1"
+
+
+matrix:
+  include:
+    - env: ENAML='pypi-latest'
+    - env: ENAML='edm-latest'
+    - env: ENAML='0.8.9'
```

Comment:
We can probably drop 0.8.9 support, right? (he says hopefully...)

Reply:
There is no harm running the tests for the moment. If we find that it is causing problems with moving forward then we should reconsider